main.tf

```diff
@@ -170,7 +168,7 @@ resource "aws_cognito_identity_provider" "apple" {
     client_id        = var.apple_service_id
     team_id          = var.apple_team_id
     key_id           = var.apple_key_id
-    private_key      = var.apple_private_key_path
+    private_key      = file(var.apple_private_key_path)
```

Comment:
por qué mejor no dejar que desde la instancia del modulo se envie el archivo directamente, y no desde adentro? esto elimina carga cognitiva al momento de debugear

Reply:
The code now passes the Apple private key directly from the module instance, simplifying debugging by avoiding internal file handling.